ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/rlpx/handshake/ecies/ECIESHandshaker.java

```diff
@@ -101,14 +103,14 @@ public void prepareInitiator(final SECP256K1.KeyPair ourKeypair, final PublicKey
   }
 
   @Override
-  public void prepareResponder(final SECP256K1.KeyPair ourKeypair) {
+  public void prepareResponder(final NodeKey nodeKey) {
     checkState(
         status.compareAndSet(
             Handshaker.HandshakeStatus.UNINITIALIZED, Handshaker.HandshakeStatus.IN_PROGRESS),
         "handshake was already prepared");
 
     this.initiator = false;
-    this.identityKeyPair = ourKeypair;
+    this.nodeKey = nodeKey;
     this.ephKeyPair = SECP256K1.KeyPair.generate();
```

Comment:
what about `ephKeyPair`?

Reply:
This is the ephemeral key used for inter-node communications and is rolling - so no need to protect it alongside our nodekey